src/mixins/legal-api-mixin.ts

```diff
@@ -23,7 +23,7 @@ export default class LegalApiMixin extends Vue {
    * Fetches a draft filing.
    * @returns a promise to return the draft filing, or null if not found
    */
-  async fetchDraft (): Promise<any> {
+  async fetchDraftIA (): Promise<any> {
     // get the draft filing from the tasks endpoint
     const url = `businesses/${this.getTempId}/filings`
```

Comment:
Uh, this isn't the tasks endpoint (per comment above)...?

I'm confused (i) why this works, and (ii) if this works, why?

Reply:
I was also confused. I deleted the comment but must have restored it when i created a new endpoint altogether. 
Not sure why the comment said tasks unless this was refactored at some point. 

But yea, will remove the comment. It's a unique request to `/filings` where there is only ever 1 response,
ie `filing` where a normal request will return a `filings` array with all completed/paid filings. 

The divergence is a result of the temp reg number as the identifier, it takes a new flow in the api!

Reply:
Ah, gotcha.

Come to think of it, Filings UI does the same thing: https://github.com/bcgov/business-filings-ui/blob/c3d2a9c87604a25e64d9cac0fd773e83c0e7a7bd/src/App.vue#L514

I generally like comments. It might be worth keeping the comment and explaining this unique case. (I will do this in Filings UI.)

Reply:
![image](https://user-images.githubusercontent.com/10002889/135533650-561e6ae6-3e9e-4f69-be2b-a5da0cf35fef.png)

Reply:
Perfect! Updated